docs/protocol.md

```diff
@@ -57,25 +57,21 @@ URL template variables replace dotted paths inside curly brackets with their cor
 
 The Message Protocol defines the the data that is used as input to individual workflow tasks. This includes:
 
-* **Ingest Metadata** - Information about the current running ingest like what time it started and the name of the workflow.
-* **Meta** - The `meta` element copied from the collection configuration. This will usually contain information about the collection like its id.
-* **Provider** - Settings specific to the current provider.
-* **Workflow Task Configuration** - Templates that specify the configurations for each of the tasks that run within a workflow.
-* **Resources** - The set of external resources accessible to the task like S3 buckets to use or table names.
-* **Exception** - A field to indicate a task failure.
+* **Cumulus Meta** - External resources accessible to the Task. Tasks should generally prefer to be passed resources explicitly in their configuration rather than looking up paths here.
+* **Meta** - Metadata taken from the collection configuration and other configuration settings. Tasks may add fields to the 'meta' object at will (in their returned output) in order to pass data to future tasks.
+* **Workflow Configuration** - Specify the configurations for each of the tasks that run within a workflow.
+* **Exception** - A field to indicate a task failure. Information in this can be used by the workflow to determine next steps.
```

Comment:
`Information in **Exception** can be used by the workflow to determine next steps.`